### PR TITLE
Added presentation flow

### PR DESCRIPTION
**Pull request by booleanbalaji**
**Event title:**

**Organizer's name:**

cc @hasgeek/events-review

### CODE REVIEW

File: _50p_events/public-lecture-zooko.md [hunk 1]
```diff
@@ -44,13 +44,24 @@ overview:
...
     **5:00 - 5:20 Flash talks by audience**
 
     **5:20 - 5:30 Networking break**
 
     **5:30 - 7:00 Lecture by Zooko Wilcox followed by Q&A**
+    
+    ## Presentation flow
+    
+    - What are blockchains and why do they matter?
+    - What are blockchains and why do they matter?
```
Comment (akshaybhalotia) [inline _50p_events/public-lecture-zooko.md]: Written twice

File: _50p_events/public-lecture-zooko.md [hunk 2]
```diff
@@ -44,13 +44,24 @@ overview:
...
+    - Brief introduction to BTC, the original promise of privacy, and why BTC in practice offers (ever-diminishing) plausible deniability and not in fact true privacy
+    - Approaches to privacy on blockchains leading up to an intro to Zk-snarks/zero knowledge protocols and the genesis of Z-Cash
+    - Z-Cash: a) origins and design, b) technical and adoption achievements to date, c) future roadmap, d) how did you build the community and how does Z-Cash support its developer community on an on-going basis
+    - Coup de Grace: What are the specific ways in which developers or technical folks can contribute to Z-Cash? How can they build applications on top of the protocol? Can you give examples of apps/projects you would like to see fructify? How can entrepreneurs/product owners integrate Z-Cash/zk-snarks. What are the benefits that this brings?
+    - Is there anything that's critical for a developer that is not on github or the website that you would like to share.
```
Comment (akshaybhalotia) [inline _50p_events/public-lecture-zooko.md]: Inconsistent language. Rest are talking points, this is a question.

**Comment by akshaybhalotia** [comment]
@booleanbalaji Please check this

File: _50p_events/public-lecture-zooko.md [hunk 3]
```diff
@@ -44,13 +44,23 @@ overview:
...
     **5:30 - 7:00 Lecture by Zooko Wilcox followed by Q&A**
+    
+    ## Presentation flow
+    
+    - What are blockchains and why do they matter?
+    - Brief introduction to BTC, the original promise of privacy, and why BTC in practice offers (ever-diminishing) plausible deniability and not in fact true privacy
+    - Approaches to privacy on blockchains leading up to an intro to Zk-snarks/zero knowledge protocols and the genesis of Z-Cash
+    - Z-Cash: a) origins and design, b) technical and adoption achievements to date, c) future roadmap, d) how did you build the community and how does Z-Cash support its developer community on an on-going basis
+    - Coup de Grace: What are the specific ways in which developers or technical folks can contribute to Z-Cash? How can they build applications on top of the protocol? Can you give examples of apps/projects you would like to see fructify? How can entrepreneurs/product owners integrate Z-Cash/zk-snarks. What are the benefits that this brings?
+    - Is there anything that's critical for a developer that is not on github or the website that you would like to share?
```
Comment (akshaybhalotia) [inline _50p_events/public-lecture-zooko.md]: This is still not right. Who is asking this question? To whom? Who is `you` here - audience, HasGeek, Zooko, somebody else?

Comment (booleanbalaji) [reply _50p_events/public-lecture-zooko.md]: Rewriting this. Got this copy from someone else.

**Comment by booleanbalaji** [comment]
@akshaybhalotia Done

**Comment by akshaybhalotia** [comment]
Attaboy

**Pull request merged**